gateway/1.4/modules/ROOT/pages/flex-review-prerequisites.adoc

```diff
@@ -70,12 +70,17 @@ Flex Gateway is designed to run in cloud-native architectures. You can therefore
 
 == Hardware Requirements
 
+A single Flex Gateway can support multiple backend APIs. To support more backend APIs, you can deploy multiple replicas or additional Flex Gateways. 
```

Comment:
```suggestion
A single Flex Gateway can support multiple backend APIs. To support more backend APIs, you can deploy multiple replicas or additional Flex Gateways. 
```
I'm not sure these are hardware requirements?

Reply:
Maybe it's equally confusing to say "the most amount of", because then they'll ask what that amount is.

So, yeah, maybe it's all good.

Reply:
Though I'm still not sure these sentences should live under "Hardware Requirements".